commands/item/trade.js

```diff
@@ -0,0 +1,146 @@
+const { Command } = require('discord.js-commando');
+
+const Currency = require('../../currency/Currency');
+const Inventory = require('../../currency/Inventory');
+const ItemGroup = require('../../currency/ItemGroup');
+
+module.exports = class ItemTradeCommand extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'item-trade',
+			aliases: [
+				'trade-items',
+				'trade-item',
+				'items-trade'
+			],
+			group: 'item',
+			memberName: 'trade',
+			description: `Trade items with another user.`,
+			guildOnly: true,
+			throttling: {
+				usages: 2,
+				duration: 3
+			},
+
+			args: [
+				{
+					key: 'member',
+					prompt: 'what user would you like to give your item(s)?\n',
+					type: 'member'
+				},
+				{
+					key: 'offerAmount',
+					label: 'amount of items to give',
+					prompt: 'how many would you like to offer?\n',
+					type: 'integer',
+					min: 1
+				},
+				{
+					key: 'offerItem',
+					prompt: 'what item would you like to offer?\n',
+					type: 'string'
+				},
+				{
+					key: 'receiveAmount',
+					label: 'amount of items to receive',
+					prompt: 'how many would you like to receive?\n',
+					type: 'integer',
+					min: 1
+				},
+				{
+					key: 'receiveItem',
+					prompt: 'what item would you like to receive?\n',
+					type: 'string'
+				}
+			]
+		});
+	}
+
+	async run(msg, args) {
```

Comment:
Sure can do. I tried to formulate it as naturally as possible. As in:
I want to `trade 5 cards` *for* `300 donuts`